includes/batch.inc

```diff
@@ -241,8 +246,18 @@ function islandora_paged_content_create_ocr_batch_operation(array $pages, array
   $page = islandora_paged_content_start_batch_operation($pages, $context);
   if ($page) {
     $page = islandora_object_load($page);
-    $context['results']['successful'][] = islandora_paged_content_page_derive_ocr_datastreams($page, $options);
-    islandora_paged_content_end_batch_operation($context, 1, array('Performed OCR on @successful of @total pages.'));
+    $ocr = islandora_paged_content_page_derive_ocr_datastreams($page, $options, FALSE);
+    foreach ($ocr as $dsid => $result) {
+      if ($result !== FALSE) {
+        $context['results']['files'][] = $result;
+        // OCR files will be required for concatenation.
+        if ($dsid == 'OCR') {
+          $context['results']['pages'][] = $result;
```

Comment:
Why not append directly to the our "consolidated" business?... Could then delete the file when exiting here, yeah? (assuming it gets updated on the page correctly)... And wouldn't have to track in the `files` array above...  Dunno?